paddle/fluid/inference/api/paddle_mkldnn_quantizer_config.h

```diff
@@ -26,10 +26,11 @@ namespace paddle {
 
 // Algorithms for finding scale of quantized Tensors.
 enum class ScaleAlgo {
-  NONE,    // Do not compute scale
-  MAX,     // Find scale based on the maximum absolute value
-  MAX_CH,  // Find scale based on the maximum absolute value per channel
-  KL,      // Find scale based on KL Divergence
+  NONE,      // Do not compute scale
+  MAX,       // Find scale based on the max absolute value
+  MAX_CH,    // Find scale based on the max absolute value per output channel
+  MAX_CH_T,  // Find scale based on the max absolute value per input channel
```

Comment:
imo `... per output channel of a transposed tensor` would convey the meaning a little bit closer

Reply:
I remember I spent quite some time figuring out how to not exceed 80 characters per line limit still maintaining the message that would suffice 😄 But yes, your description is definitely much more accurate. I'll modify it.